### PR TITLE
api: add setDebugAPILimiter api at admin namespace

### PR DESCRIPTION
**Pull request by yoomee1313**
## Proposed changes

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: node/cn/api.go [hunk 1]
```diff
@@ -271,6 +273,32 @@ func (api *PrivateAdminAPI) GetSpamThrottlerCandidateList(ctx context.Context) (
...
+}
+
+func (api *PrivateAdminAPI) SetDebugAPILimiter(ctx context.Context, heavyAPILimit int32, duration string) (bool, error) {
+	if !api.cn.config.DisableUnsafeDebug {
+		return false, errors.New("unsafe-debug is enabled")
+	}
+	timeLimit, err := time.ParseDuration(duration)
+	wrongDuration := err != nil || timeLimit <= time.Duration(0)
+
+	if heavyAPILimit <= 0 && wrongDuration {
```
Comment (hyunsooda) [inline node/cn/api.go:290]: How about proceeding with the suitable value? Presently, the API requires updates for both values even if an operator intends to modify only one. Alternatively, you could consider splitting the function into two separate ones for intuitive.

**Review by hyunsooda — COMMENTED** [review]
With this additional logic, the invoke of decrementing one seems to be reconsidered together. I'm concerned about this for the following scenario.

Initially, an operator employs the default value of `500` for the concurrency limit.
Subsequently, the operator adjusts the value to `10` using the new API.
During this period, let's assume there are `500` scheduled code to be excuted `defer atomic.AddInt32(&heavyAPIRequestCount, -1)`.
Consequently, the count reaches `-490`. Given that the limiter intervenes at the threshold of `10`, the boundary of `500` requests persists, as it was before.

**Comment by yoomee1313** [comment]
> Consequently, the count reaches -490.
defer -1 is executed only if the count value was increased. So, negative value seems not feasible. Btw, I have tested the simple load test, and checked the value did not go down zero.

**Comment by hyunsooda** [comment]
I'm sorry. I confused two variables. Now I see that the negative value is infeasible.

**Comment by yoomee1313** [comment]
https://github.com/klaytn/klaytn/pull/1996 succeeds this PR.

**Comment by hyunsooda** [comment]
@yoomee1313,

> https://github.com/klaytn/klaytn/pull/1996 succeeds this PR.

The PR 1996 does not address this setter/getter. A property `api.cn.config.StateRegenerationTimeLimit` can be displayed with the PR 1996. Do you mean that the PR would shepherd this issue as well?

**Comment by yoomee1313** [comment]
@hyunsooda For now, only getter is needed.

**Pull request closed without merging**